### ISSUE TITLE
duden.de: detection

### ISSUE THREAD

**Issue by bgeron** [uBlockOrigin/uAssets#14540]
### Prerequisites

- [X] I read and understood the [policy as to what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets)
- [X] I verified that the issue has not already been reported (use [this button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out)
- [X] I forced an update of my filter lists (**how to do this:** [click _"Purge all caches"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches), then [click _"Update now"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now))
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I verified that the issue is not caused by removing any of the default lists
- [X] I did not enable external filter lists, or I verified that the issue still occurs without enabling external filter lists
- [X] I do not have custom filters/rules, or I verified that the issue still occurs without custom filters/rules
- [X] I am not using [another content blocker along uBO](https://twitter.com/gorhill/status/1033706103782170625) (also mind browser built-in blockers)
- [ ] I did not answer truthfully to **all** the above checkpoints

### URL address of the web page

`https://www.duden.de/`

### Category

detection

### Describe the issue

Site is asking for ad blocker to be disabled

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![image](https://user-images.githubusercontent.com/950328/186970624-d7773f0e-7865-44f7-b583-c3632033fb0c.png)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.44.0
Firefox: 103
filterset (summary): 
  network: 82890
  cosmetic: 45110
  scriptlet: 17691
  html: 734
listset (total-discarded, last updated): 
  default: 
    user-filters: 0-0, never
    ublock-filters: 33205-35, 3d.21h.33m
    ublock-badware: 4218-0, 3d.21h.32m
    ublock-privacy: 263-0, 1d.21h.42m
    ublock-abuse: 76-0, 1d.21h.41m
    ublock-unbreak: 1850-0, 3d.21h.31m
    easylist: 69389-599, 3d.21h.30m
    easyprivacy: 27403-578, 3d.21h.34m
    urlhaus-1: 7384-0, 6h.14m
    plowe-0: 3633-3, 1d.21h.43m
    ublock-quick-fixes: 389-0, 6h.15m
filterset (user): [empty]
trustedset: 
  added: [array of 10 redacted]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 3127 ms (selfie)
  maxAssetCacheWait: 161 ms
popupPanel: 
  blocked: 3
  blockedDetails: 
    duden.de: 1
    doubleclick.net: 1
    googlesyndication.com: 1
```
</details>

**Comment by okiehsch** [comment]
How is "first-level-support" now part of a german dictionary. 🤔 

It's a firefox only issue. I will add a fix.

**Comment by MasterKia** [comment]
@okiehsch 
```
||duden.de/themes/custom/numero/component/anti-adblocker^
duden.de###aab-container
```

**Comment by MasterKia** [comment]
I think the old scriptlets aren't necessary anymore.

**Comment by bgeron** [comment]
Works — amazing, thank you!

**Comment by okiehsch** [comment]
Kein Problem.

@MasterKia Yes, I can't reproduce any ad reinsertion anymore, I will remove them.

**Comment by okiehsch** [comment]
@MasterKia 
`duden.de##+js(aopr,showAdBlockerOverlay)`
works for me, better than blocking the first-party request imo.

**Comment by MasterKia** [comment]
> better than blocking the first-party request imo.

Ok, I thought network filters were prefered over scriptlets.

So this way we are letting the anti-adb load then we defuse it.

**Comment by okiehsch** [comment]
> So this way we are letting the anti-adb load then we defuse it.

The idea is that script looks like a generic adblock solution, it's easier for the site to change the name of the script than the script itself.
You can block the script as well if you like, it's just my preference it's not a set in stone rule.

**Comment by MasterKia** [comment]
> it's easier for the site to change the name of the script than the script itself.

Well I figured changing the function name should be easy too.

> You can block the script as well if you like, it's just my preference it's not a set in stone rule

I understand. Let's see how they react.

**Comment by akostem** [comment]
>It's a firefox only issue. I will add a fix.

I also have this problem on Vivaldi. The browser's own adblocker is turned off.

**Comment by MasterKia** [comment]
@akostem It has been fixed 20 hours ago.
Force update your filter lists and test again:
https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches

**Comment by akostem** [comment]
@MasterKia I believe to have done the steps required, and as it had no effect on the problem, I reinstalled the extension. I made a few browser restarts in between, but in the end, the issue persists. @okiehsch said earlier that it is a firefox only issue, and commit [#14540](https://github.com/uBlockOrigin/uAssets/commit/716f5c84d1c23087eaed5a3e7f50cc26b9f64774) seems to only change something on firefox. (Although I do not fully understand the syntax.)
